src/main/java/core/basesyntax/models/FigureSupplier.java

```diff
@@ -0,0 +1,99 @@
+package core.basesyntax.models;
+
+import core.basesyntax.models.painter.Color;
+import core.basesyntax.models.painter.ColorSupplier;
+import java.util.Random;
+
+public class FigureSupplier {
+    private static final double PROPERTY_MULTIPLIER = 10;
+    private static final Color DEFAULT_COLOR = Color.WHITE;
+    private static final Figures DEFAULT_FIGURE = Figures.CIRCLE;
+    private final Random random = new Random();
+    private final ColorSupplier colorSupplier = new ColorSupplier(random);
+
+    public Figure getDefaultFigure() {
+        return createFigure(DEFAULT_FIGURE);
+    }
+
+    public Figure getRandomFigure() {
+        int randFigureIndex = random.nextInt(Figures.values().length);
+        Figure figure = createFigure(Figures.values()[randFigureIndex]);
+        return randomizeProperties(figure);
+    }
+
+    private Figure createFigure(Figures newFigure) {
+        switch (newFigure) {
+            case CIRCLE:
+                return createCircle();
```

Comment:
can we write here something like `return new Circle(randomColor, randomRadius);` for example?

Reply:
Sure. Recognized it now.
+fixed in new commit